pkg/envoy/ads/response.go

```diff
@@ -65,7 +65,7 @@ func (s *Server) sendResponse(typeURIsToSend mapset.Set,
 
 		// Handle request when is not provided, and the SDS case
 		var finalReq *xds_discovery.DiscoveryRequest
-		if request == nil {
+		if request == nil || request.TypeUrl == string(envoy.TypeWildcard) {
```

Comment:
Would change `string(envy.TypeWildcard)` to `envoy.TypeWildcard.String()` and add `String()` method receiver for the type.

Reply:
can do. Not that I remember, I think that was the VC autocompletion